### PR TITLE
ShuffleArray.md

### PR DESCRIPTION
**Pull request by void-57**

### CODE REVIEW

**Review by DevTMK — CHANGES_REQUESTED** [review]
@void-57 You are not supposed to remove template.md file
and move the snip file to [`src`](https://github.com/DevTMK/snips/blob/main/src) folder

File: ShuffleArray.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# [Shuffle Array]
+
```
Comment (DevTMK) [inline ShuffleArray.md:2]: Add short description here

**Comment by void-57** [comment]
oh sorry i will do again

File: ShuffleArray.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+```
+  shuffle([1, 2, 3, 4]); ---> [2, 4, 3, 1]
+```
+
+## Snip Langauge
+* [C++](#c++)
+
+### C++
+```c++
+  #include <bits/stdc++.h>
```
Comment (DevTMK) [inline ShuffleArray.md:13]: See if you can do it without any external dependancy

File: ShuffleArray.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ 
+    unsigned seed = 0;
+    shuffle(arr, arr + n,default_random_engine(seed));
+     cout<<"\nThe Shuffled Array is: \n";
+    for (int i = 0; i < n; ++i)
+        cout << arr[i] << " ";
+}
+ 
+
+int main()
```
Comment (DevTMK) [inline ShuffleArray.md:28]: Just keep the shuffle function

**Review by DevTMK — CHANGES_REQUESTED** [review]

File: ShuffleArray.md (hunk 3 above)
Comment (void-57) [reply ShuffleArray.md:28]: okk

**Comment by DevTMK** [comment]
It's been a long, there are no changes. Closing this PR for now

**Pull request closed without merging**